modules/gdscript/gdscript_editor.cpp

```diff
@@ -65,7 +65,7 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 					   "%TS%pass\n\n" +
 					   "#func _process(delta):\n" +
 					   "#%TS%# Called every frame. Delta is time since last frame.\n" +
-					   "#%TS%# Update game logic here.\n" +
+					   "#%TS%# Update game logic here\n" +
```

Comment:
That makes no sense, the sentence above ends with a period.
The PEP8 convention is (at least as I understand it and saw it applied):
- single line comment, no period
- multiple lines/multiple sentences: all sentences end with a period

Reply:
Sorry, I understood the guide incorrectly. I will just add the missing periods. I just wish that some care for the consistency is taken, and the docs (specially the ones inside the editor) follow suit.